app/hub/Views/TutorialView/TutorialViewReducer.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Reducer used throughout the Tutorial View's flow
+ *
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2018 Ghostery, Inc. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+const initialState = {};
+
+function TutorialViewReducer(state = initialState, action) {
+	switch (action.type) {
+		// Tutorial View
+		case 'INIT_TUTORIAL_PROPS': {
+			const {
+				activeIndex,
+				hrefPrev,
+				hrefNext,
+				hrefDone,
+				textPrev,
+				textNext,
+				textDone,
+			} = action.data.navigation;
+			return Object.assign({}, state, {
+				tutorial: {
+					navigation: {
+						activeIndex,
+						hrefPrev,
+						hrefNext,
+						hrefDone,
+						textPrev,
+						textNext,
+						textDone,
+					}
+				},
+			});
+		}
+		case 'SET_TUTORIAL_NAVIGATION': {
+			const {
+				activeIndex,
+				hrefPrev,
+				hrefNext,
+				hrefDone,
+				textPrev,
+				textNext,
+				textDone,
+			} = action.data;
+			return Object.assign({}, state, {
+				tutorial: Object.assign({}, state.tutorial, {
+					navigation: {
+						activeIndex,
+						hrefPrev,
+						hrefNext,
+						hrefDone,
+						textPrev,
+						textNext,
+						textDone,
+					},
+				}),
+			});
+		}
+
+		default: return state;
+	}
+}
+
+export default TutorialViewReducer;
```

Comment:
@trickpattyFH20 all `case` strings here should be `const` imported from a constants file.